guides/common/modules/proc_configuring-the-mcp-server-for-project.adoc

```diff
@@ -12,7 +12,8 @@ Perform these steps on the same system where your MCP client will run.
 ifdef::satellite[]
 * You are logged in to the registry.redhat.io container registry.
 endif::[]
-* The CA bundle for your {Project} is available on the system.
+* The CA bundle for your {Project} is available on your system.
+You can download it from `https://_{foreman-example-com}_/unattended/public/foreman_raw_ca`.
```

Comment:
_non-blocking:_
Installing standalone hammer also has this as a prerequisite https://github.com/theforeman/foreman-documentation/blob/cddb86f4df06266d752c7d50771031cc0097e9d9/guides/common/modules/proc_installing-standalone-hammer.adoc?plain=1#L11 . Now that we figured out a way how to actually get it, should we mention it there as well?